index.js

```diff
@@ -0,0 +1,62 @@
+const mongoose = require("mongoose");
+const Person = require("./Person.js");
+
+mongoose.connect("mongodb://localhost/contacts");
+
+Person.create([
+  {
+    name: "Mohammed",
+    age: 25,
+    favoriteFoods: ["pizza", "apple"],
+  },
+  {
+    name: "Hoda",
+    age: 28,
+    favoriteFoods: ["burger", "watermelon"],
+  },
+  {
+    name: "Baliegh",
+    age: 25,
+    favoriteFoods: ["feteer", "oranges"],
+  },
+]);
+//then((person) => console.log(person));
+
+//find all people with a given name
+// Person.find({ name: "Mohammed" }).then((people) => console.log(people));
+
+//find one person with a given favorite food
+// Person.findOne({ favoriteFoods: "pizza" }).then((people) =>
```

Comment:
`findOne` method only returns one document, thus it would be better in this case to call the param `person` instead of `people`.